dicom_util/src/main/java/com/google/cloud/healthcare/DicomWebClientJetty.java

```diff
@@ -74,7 +74,7 @@ public void stowRs(InputStream in) throws DicomWebException {
       FuturePromise<Session> sessionPromise = new FuturePromise<>();
       client.connect(sslContextFactory, new InetSocketAddress(uri.getHost(), CONNECT_PORT),
           new ServerSessionListener.Adapter(), sessionPromise);
-      Session session = sessionPromise.get(5, TimeUnit.SECONDS);
+      Session session = sessionPromise.get(600, TimeUnit.SECONDS);
```

Comment:
Can you add a comment with a link to an issue that describes why this timeout has to be set so high?

Reply:
done